test/e2e/resource_placement_hub_workload_test.go

```diff
@@ -105,9 +109,30 @@ var _ = Describe("placing workloads using a CRP with PickAll policy", Label("res
 				Name:      testJob.Name,
 				Namespace: workNamespace.Name,
 			},
+			{
+				Group:     "apps",
+				Version:   "v1",
+				Kind:      "StatefulSet",
+				Name:      testStatefulSet.Name,
+				Namespace: workNamespace.Name,
+			},
+			// PVCs created by StatefulSet controller from volumeClaimTemplates
+			// Kubernetes StatefulSet controller uses naming convention: <volumeClaimTemplate-name>-<statefulset-name>-<replica-index>
+			{
+				Version:   "v1",
+				Kind:      "PersistentVolumeClaim",
+				Name:      fmt.Sprintf("%s-%s-%d", testStatefulSet.Spec.VolumeClaimTemplates[0].Name, testStatefulSet.Name, 0),
+				Namespace: workNamespace.Name,
+			},
+			{
+				Version:   "v1",
+				Kind:      "PersistentVolumeClaim",
+				Name:      fmt.Sprintf("%s-%s-%d", testStatefulSet.Spec.VolumeClaimTemplates[0].Name, testStatefulSet.Name, 1),
+				Namespace: workNamespace.Name,
+			},
 		}
 		// Use customizedPlacementStatusUpdatedActual with resourceIsTrackable=false
-		// because Jobs don't have availability tracking like Deployments/DaemonSets do
+		// because Jobs and PVCs don't have availability tracking like Deployments/DaemonSets do
```

Comment:
PVC has conditions, I wonder why it's not trackable? However, do we really want to place PVC if they are the by-product of a statefulSet?